src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -685,22 +685,21 @@
 helps['role assignment create'] = """
 type: command
 short-summary: Create a new role assignment for a user, group, or service principal.
+long-summary: In a future release, --scope argument will become required for creating a role assignment. Please explicitly specify --scope.
 examples:
-  - name: Create role assignment for an assignee.
-    text: az role assignment create --assignee sp_name --role a_role
+  - name: Create role assignment to grant an assignee Reader role on a virtual machine.
+    text: az role assignment create --assignee sp_name --role Reader --scope /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyResourceGroup/providers/Microsoft.Compute/virtualMachines/MyVm
   - name: Create role assignment for an assignee with description and condition.
     text: >-
-        az role assignment create --role "Owner" --assignee "John.Doe@Contoso.com"
+        az role assignment create --role Owner
+        --scope /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyResourceGroup/providers/Microsoft.Storage/storageAccounts/MyStorageAccount
+        --assignee "John.Doe@Contoso.com"
         --description "Role assignment foo to check on bar"
         --condition "@Resource[Microsoft.Storage/storageAccounts/blobServices/containers:Name] stringEquals 'foo'"
         --condition-version "2.0"
     supported-profiles: latest
-  - name: Create a new role assignment for a user, group, or service principal. (autogenerated)
```

Comment:
@jiasli , in removing this autogenerated line, won't it just come back in again in the next Aladdin PR ([example](https://github.com/Azure/azure-cli/pull/25137)) that backfills the short and long descriptions, and examples in _help.py?

Reply:
Based on my understanding, Aladdin-generated content is merged into `_help.py` on-demand and won't be automatically done.

Reply:
@jiasli, "on demand" = someone in engineering is reviewing each Aladdin PR change and accepting/manually implementing(?) it?  I have a lot of questions on the Aladdin process, and this really isn't the place so as long as your updates won't be over-written, we can move on.

Reply:
Correct. Someone needs to manually accept it.